File: data/sample/atcoder/abc100_a/Go/impl_02.go
package main

import (
	"fmt"
)

func main() {
	var a int
	var b int
	fmt.Scan(&a, &b)
	//となりあわない
	if a <= 8 && b <= 8 {

		fmt.Print("Yay!")
	} else {
		fmt.Print(":(")
	}
}

File: data/sample/atcoder/abc100_a/Rust/impl_02.rs
#[allow(unused_imports)]
use proconio::{input, marker::Chars};
fn main() {
    input! {
        a: u8,
        b: u8
    }
    let res = if a <= 8 && b <= 8 { "Yay!" } else { ":(" };
    println!("{}", res);
}
